interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -30,11 +30,11 @@ abstract class CatsPlatform extends CatsInstances {
 }
 
 abstract class CatsInstances extends CatsInstances1 {
-  implicit def ioContextShift[E]: ContextShift[IO[E, ?]] = new ContextShift[IO[E, ?]] {
-    override def shift: IO[E, Unit] =
-      IO.yieldNow
+  implicit def ioContextShift[R]: ContextShift[TaskR[R, ?]] = new ContextShift[TaskR[R, ?]] {
+    override def shift: TaskR[R, Unit] =
+      ZIO.yieldNow
```

Comment:
Once you add the companion object, you can change all these methods to use `TaskR` companion object.

Reply:
`ContextShift` should be available for `ZIO[R, E, A]`